### ISSUE TITLE
Latest Puppeteer Chromium version and compatibility with Chrome

### ISSUE THREAD

**Issue by parvathi** [puppeteer/puppeteer#3163]
Hi,

I have posted following question today and I got a response saying Puppeteer 1.7.0 latest version uses the Chromium 70. But package.json says chromium version is 587164. Can you please confirm it is using 70 and just a mistake in package.json? 

Also would the latest code work with Chrome 69 or 70?

https://github.com/GoogleChrome/puppeteer/blob/master/package.json
"puppeteer": {
    "chromium_revision": "587164"
  },

***********************************************
We are trying to use the latest version of puppeteer which is using Chromium version 58. We are trying to install in cloud foundry but some x server files are missing and we are having to manually deploy this. It looks like Chrome 59 doesn't depend on display server. If we use Chrome version 59 with puppeteer instead of chromium that comes with it, is it guaranteed to work? Or is there a plan for Chromium to get away from display server?

thanks
Parvathi

**Comment by aslushnikov** [comment]
> Can you please confirm it is using 70 and just a mistake in package.json?

Hi @parvathi, there's no mistake in package.json. The number you see is a *revision* number, which
has nothing to do with *Chromium version*. Every release we publish the version of bundled chromium, e.g. see our release notes for 1.7.0: [release notes](https://github.com/GoogleChrome/puppeteer/releases/tag/v1.7.0)

You can also convert *revision number* to *Chromium version* manually, using https://omahaproxy.appspot.com/ and filling in the "find release" section of the site:

![image](https://user-images.githubusercontent.com/746130/44883266-70b50a80-acae-11e8-9157-4447db39ca6a.png)

> Also would the latest code work with Chrome 69 or 70?

It will work with Chrome 70.

Hope this helps!

**Comment by bluepeter** [comment]
Hi @aslushnikov this technique to find the version number isn't working for the latest listed revision number 591618. [I get](https://storage.googleapis.com/chromium-find-releases-static/2b0.html#2b066379565016a82e3da551c65e9856379229b2):

> No data found. If a valid commit was specified this means the commit is not yet in a release.

Any tips to find the ver number for this revision?

**Comment by jzlocz** [comment]
Changing chromium_revision without changing package version by maintainers of puppeteer can broke deployments where chrome version is fixed and installed manually.

This is the case: https://github.com/GoogleChrome/puppeteer/commit/4a53bca6b081643855e005c940d8cbc52404e99e
I'm installing puppeteer@1.1.1 and my chrome is no longer compatible. 

It's hard to follow you. Do you have plan to change package version in the future or I have to handle this problem by my self?
